views/js/qtiCreator/editor/ckEditor/htmlEditor.js

```diff
@@ -90,6 +94,10 @@ define([
             });
         }
 
+        if(languagePluginEnabled) {
+            removePlugins.push('language');
```

Comment:
**thought:** I wonder if it would not be easier/better to do the opposite, say add the plugin when needed instead of removing when not needed.

Reply:
disabling plugin is a bit easier if checking the file we have already a flow disabling plugins, and it may be better for future when we will have only solar (enabled by default)